com.unity.mobile.notifications/Editor/AndroidNotificationResourcesPostProcessor.cs

```diff
@@ -1,142 +1,216 @@
+#if UNITY_EDITOR && PLATFORM_ANDROID
```

Comment:
Could we change PLATFORM_ANDROID->UNITY_ANDROID
since PLATFORM_ANDROID is not documented

Reply:
We can, I just didn't touch that... 
And we can remove UNITY_EDITOR as it's under Editor folder already.

Reply:
Fxied at https://github.com/Unity-Technologies/com.unity.mobile.notifications/pull/8/commits/508ace7521feba484ac46d15408b7810ebe73029